strawberry_django_plus/optimizer.py

```diff
@@ -55,6 +54,13 @@
 )
 from .utils.typing import TypeOrSequence
 
+try:
+    from django.contrib.contenttypes.fields import GenericForeignKey, GenericRelation
+except (ImportError, RuntimeError):  # pragma:nocover
+    GenericForeignKey = type(None)
+    GenericRelation = type(None)
```

Comment:
Just `None` is enough because you will usually want to test this with `GenericForeignKey is not None`

```suggestion
    GenericForeignKey = None
    GenericRelation = None
```

Reply:
required for isinstance

Reply:
You don't want to do `isinstance` if the name is `None`.

As I suggested, you can change a code like this:

```python
if isinstance(foobar, GenericForeignKey):
    ...
```

To:

```python
if GenericForeignKey is not None and isinstance(foobar, GenericForeignKey):
    ...
```

Then you can use `None` directly